tests/IntegrationTest.php

```diff
@@ -40,4 +42,34 @@ protected static function getTempFile()
     {
         return self::getFixturesDir().DIRECTORY_SEPARATOR.'.tmp.php';
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected static function createIntegrationCaseFactory()
+    {
+        return new InternalIntegrationCaseFactory();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected static function assertRevertedOrderFixing(IntegrationCase $case, $fixedInputCode, $fixedInputCodeWithReversedFixers)
+    {
+        parent::assertRevertedOrderFixing($case, $fixedInputCode, $fixedInputCodeWithReversedFixers);
+
+        $settings = $case->getSettings();
+
+        if (!isset($settings['isExplicitPriorityCheck'])) {
+            static::markTestIncomplete('Missing `isExplicitPriorityCheck` extension setting.');
+        }
+
+        if ($settings['isExplicitPriorityCheck']) {
+            static::assertSame(
+                $settings['isExplicitPriorityCheck'],
```

Comment:
this is always `true`-ish here, because of line 67

Reply:
done